tests/SiteConfig/LoginFormAuthenticatorTest.php

```diff
@@ -183,4 +183,62 @@ public function testLoginWithGoodSiteConfigNotLoggedInData()
 
         $this->assertTrue($loginRequired);
     }
+
+    public function testLoginPostWithUserAgentHeaderWithData()
+    {
+        $siteConfig = new SiteConfig([
+            'host' => 'nextinpact.com',
+            'loginUri' => 'https://compte.nextinpact.com/Account/Login',
+            'usernameField' => 'UserName',
+            'passwordField' => 'Password',
+            'username' => 'johndoe',
+            'password' => 'unkn0wn',
+            'httpHeaders' => [
+                'HTTP_user-agent' => 'Wallabag (Guzzle/5)',
```

Comment:
by moving to LoginFormAuthenticator this become
```suggestion
                'user-agent' => 'Wallabag (Guzzle/5)',
```